src/main/resources/db/changelog/changes/create-user-tenant-table.xml

```diff
@@ -7,15 +7,18 @@
   <changeSet id="MODFISTO-400@@create-user-tenant-table" author="azizbekxm">
     <createTable tableName="user_tenant">
 
```

Comment:
We will have merge confliscts, firstly we need to merge PR https://github.com/folio-org/mod-consortia/pull/8
and pull from master and remove these changes with schema

Reply:
done